pandas/core/frame.py

```diff
@@ -3868,10 +3862,6 @@ def isetitem(self, loc, value) -> None:
         self._iset_item_mgr(loc, arraylike, inplace=False)
 
     def __setitem__(self, key, value):
-        if not PYPY and using_copy_on_write():
-            if sys.getrefcount(self) <= 3:
-                raise ChainedAssignmentError(_chained_assignment_msg)
-
```

Comment:
Why is this changing?

Reply:
I'm not sure why this happened except that it happened when I did an upstream merge main in to my branch.